app/src/main/java/agency/five/codebase/android/movieapp/ui/component/ActorCard.kt

```diff
@@ -0,0 +1,77 @@
+package agency.five.codebase.android.movieapp.ui.component
+
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Card
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import coil.compose.AsyncImage
+import agency.five.codebase.android.movieapp.mock.MoviesMock.getActor
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.unit.Dp
+import agency.five.codebase.android.movieapp.ui.theme.Typography
+import androidx.compose.ui.text.style.TextOverflow
+
+data class ActorCardViewState(
+    val imageUrl: String?,
+    val name: String,
+    val character: String,
+)
+
+@Composable
+fun ActorCard(
+    actorCardViewState: ActorCardViewState,
+    modifier: Modifier = Modifier,
+) {
+    Card(
+        modifier = modifier,
```

Comment:
Removed size and padding from the composable function so they can be changed from the outside.